src/shared/components/msktable/MSKTable.tsx

```diff
@@ -397,6 +403,23 @@ export default class MSKTable<T> extends React.Component<MSKTableProps<T>, {}> {
                 headers={this.store.headers}
                 rows={this.store.rows}
             />
+
+            <If condition={this.store.showingAllRows === false}>
+            <ButtonToolbar style={{marginLeft:0}} className='text-center'>
+                <PaginationControls
```

Comment:
Point #1: done.

For #2, I used to be religious about avoiding inline styles, but now I think that small pixel pushing stuff should be inline.  Otherwise you end up bloating style sheets, having to come up with all kinds of names and worry about uniqueness.  It's more mentally efficient to put it inline.  Though we wouldn't want this for an element that gets repeated many times, e.g. table cell.